automation/taskcluster/rename_snapshot_artifacts.py

```diff
@@ -16,7 +16,7 @@
 import sys
 
 
-AAR_EXTENSIONS = ('.aar', '.pom', '-sources.jar')
+ARTIFACT_EXTENSIONS = ('.aar', '.pom', '-sources.jar', '.jar')
```

Comment:
Good call on updating this constant. I hadn't thought of it! As a followup, if would be good for this file to import `taskcluster/ac_taskgraph/build_config.py` so the logic is factorized. No need to do it in this PR.